### DISCUSSION TITLE
How to extract texts between two coordinates in a page?

### DISCUSSION THREAD

**Discussion by StephenZKCurry** [jsvine/pdfplumber#1216] [Q&A]
I want to extract texts between two coordinates on a page use the PDF's underlying flow of characters as a guide for ordering and segmenting the words, rather than presorting the characters by x/y position, like argument `use_text_flow` in `extract_words()`, How can I  do that?

**Comment by jsvine** [comment]
Thanks for your interest in `pdfplumber`. I'm not sure I quite understand the question, though. Could you explain a bit more about what it would mean to "extract texts between two coordinates" (i.e., based on position) while using "the PDF's underlying flow of characters" (i.e., instead of primarily based on position)? 

Or perhaps you can provide an example of what you've been trying and what's not quite working?
